src/navigation/BottomTab/DashboardStack.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
Next several file changes are due to renaming the folder structure from drawers to bottomtabs